app/test_engine/models/test_case.py

```diff
@@ -279,6 +281,11 @@ def next_step(self) -> None:
         self.current_test_step_index += 1
         self.current_test_step.mark_as_executing()
 
+    def __print_log_separator(self) -> None:
+        logger.info(
+            "=========================================================================="
+        )
+
```

Comment:
```suggestion
        logger.info("=" * 80)

```

Could you check if this works?

Reply:
Done :)